src/Sfa.Tl.ResultsAndCertification.Web/Controllers/AdminPostResultsController.cs

```diff
@@ -385,15 +387,17 @@ private async Task<IActionResult> PostAdminPostResultAsync<T>(T model, int regis
         #region Review Change romm
         [HttpGet]
         [Route("admin/add-romm-outcome-change-grade-review-changes-core", Name = RouteConstants.AdminReviewChangesRommOutcomeCore)]
-        public async Task<IActionResult> AdminReviewChangesRommOutcomeCoreAsync()
+        public async Task<IActionResult> AdminReviewChangesRommOutcomeCoreAsync(bool isSameGrade)
         {
-            var cachedModel = await _cacheService.GetAsync<AdminAddRommOutcomeChangeGradeCoreViewModel>(CacheKey);
-            if (cachedModel == null)
+            var cachedModel =  await _cacheService.GetAsync<AdminAddRommOutcomeChangeGradeCoreViewModel>(CacheKey);
+            var cachedModel1 =  await _cacheService.GetAsync<AdminAddCoreRommOutcomeViewModel>(CacheKey);
```

Comment:
This can be done into 2 different endpoints. The variable cechedModel & cacchedModel1 could be named better.

Reply:
updated the names. This would be better option else we need to create  a view and map accordingly for a different option